docs/md/docker.md

```diff
@@ -63,39 +57,31 @@ alter user 'lychee' identified with mysql_native_password by '<your password>';
 
 ### Run with Docker Compose
 
-Change the environment variables in the [provided example](https://github.com/LycheeOrg/Lychee-Laravel-Docker/docker-compose.yml) to reflect your database credentials.
+Change the environment variables in the [provided example](./docker-compose.yml) to reflect your database credentials.
 
 Note that in order to avoid writing credentials directly into the file, you can create a `db_secrets.env` and use the `env_file` directive (see the [docs](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option)).
 
 ## Available environment variables and defaults
 
-If you do not provide environment variables or `.env` file, the [example .env file](https://github.com/LycheeOrg/Lychee-Laravel/blob/master/.env.example) will be used with some values already set by default.
+If you do not provide environment variables or `.env` file, the [example .env file](https://github.com/LycheeOrg/Lychee/blob/master/.env.example) will be used with some values already set by default.
 
 Some variables are specific to Docker, and the default values are :
 
-* PUID=1000  
-* PGID=1000  
-* USER=lychee
-* PHP_TZ=America/New_York
+* `PUID=1000`
+* `PGID=1000`
+* `USER=lychee`
+* `PHP_TZ=UTC`
+* `STARTUP_DELAY=0`
 
 ## Advanced configuration
 
-Note that nginx will accept by default images up to 100MB (`client_max_body_size 100M`) and that PHP parameters are overridden according to the [recommendations of the FAQ](faq.html#i-cant-upload-photos).
+Note that nginx will accept by default images up to 100MB (`client_max_body_size 100M`) and that PHP parameters are overridden according to the [recommendations of the Lychee FAQ](https://lycheeorg.github.io/docs/faq.html#i-cant-upload-photos).
 
 You may still want to further customize PHP configuration. The first method is to mount a custom `php.ini` to `/etc/php/7.3/fpm/php.ini` when starting the container. However, this method is kind of brutal as it will override all parameters.
 
 Instead, we recommend to use the `PHP_VALUE` directive of PHP-FPM to override specific parameters. To do so, you will need to mount a custom `nginx.conf` in your container :
 
-* 1 &mdash; Take the [default.conf](https://github.com/LycheeOrg/Lychee-Laravel-Docker/default.conf) file as a base
+* 1 &mdash; Take the [default.conf](./default.conf) file as a base
```

Comment:
you sure on that one ?

Reply:
Good spot!

Reply:
Those were like that before - you must have changed them for v1 of the docs. Those aren't mine lol